tools/PI/DevHome.PI/ViewModels/AppDetailsPageViewModel.cs

```diff
@@ -63,51 +62,15 @@ public void UpdateTargetProcess(Process process)
                     {
                         AppInfo.MainModuleFileName = targetProcess.MainModule.FileName;
                         uint binaryTypeValue;
+
+                        // TODO GetBinaryType only distinguishes x86 from x64. It doesn't allow for ARM or ARM64.
```

Comment:
Does DevHome have an issue tracking this potential work?

Reply:
Yes.